src/System.Runtime.Extensions/src/System/IO/Path.cs

```diff
@@ -49,13 +49,13 @@ public static class Path
 #endif // !PLATFORM_UNIX
 
         // Platform specific alternate directory separator character.  
-        // This is backslash ('\') on Unix, and slash ('/') on Windows 
-        // and MacOS.
+        // This is slash ('/') on Windows and MacOS.
+        // There is no alternate separator on Unix, so the normal separator is used.
         // 
 #if !PLATFORM_UNIX        
         public static readonly char AltDirectorySeparatorChar = '/';
 #else
-        public static readonly char AltDirectorySeparatorChar = '\\';
+        public static readonly char AltDirectorySeparatorChar = '/';
 #endif // !PLATFORM_UNIX
```

Comment:
Given the comment above, is it necessary to have preprocessor directive at all? 
IMO, this can be trimmed down to:

```c#
// This is backslash ('/') on Windows and MacOS.
// There is no alternate separator on Unix, so the default separator ('/') is used.
//
public static readonly char AltDirectorySeparatorChar = '/';
```